.github/actions/vercel_DR_publish/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Deploy to Vercel
+description: Deploy to Vercel
+inputs:
+  VERCEL_ORG_ID:
+    description: Vercel organization ID
+    required: true
+  VERCEL_PROJECT_ID:
+    description: Vercel project ID
+    required: true
+  VERCEL_TOKEN:
+    description: Vercel token
+    required: true
+  ENVIRONMENT:
+    description: Environment
+    required: true
+  VERCEL_SCOPE:
+    description: Vercel scope
+    required: true
+  ALIAS_DOMAIN_PATH:
+    description: Alias domain
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Install Vercel CLI
+      run: npm install -g vercel
+      shell: bash
+
+    - name: Pull Vercel project
+      shell: bash
+      run: |
+        vercel pull --yes --token=${{ inputs.VERCEL_TOKEN }} --scope=${{ inputs.VERCEL_SCOPE }}
```

Comment:
we should use inputs directly as per the security team's recommendation. Can we set these in env before we use it?

Reply:
this is done